src/panels/lovelace/cards/hui-picture-card.js

```diff
@@ -14,6 +14,7 @@ class HuiPictureCard extends NavigateMixin(PolymerElement) {
       <style>
         ha-card {
           overflow: hidden;
+          max-height: 100%;
```

Comment:
Why is this? This means that it will never expand past the screen height but that might be what the person wants? (especially if it's rendered as a panel)